code/unit_tests/icon_tests.dm

```diff
@@ -2,8 +2,12 @@
 	name = "ICON STATE template"
 	template = /datum/unit_test/icon_test
 
+/datum/unit_test/icon_test/New()
+	name = "ICON STATE - " + name
```

Comment:
Would it be an issue to do `"ICON STATE - [name]"` instead? I am increasingly suspicious of string additions, though admittedly since this is runtime not compile time it probably isn't subject to the same problems.